Generation/options/pileup_hepmcreader.py

```diff
@@ -0,0 +1,38 @@
+from Gaudi.Configuration import *
+
+from Configurables import ApplicationMgr, FCCDataSvc
+
+albersevent   = FCCDataSvc("EventDataSvc")
+# reads HepMC text file and write the HepMC::GenEvent to the data service
+from Configurables import HepMCReader, HepMCDumper, PoissonPileUp, HepMCFileReader, FlatSmearVertex
+
+# assuming mm
+smeartool = FlatSmearVertex(
+    xVertexMin = -0.5,
+    xVertexMax = 0.5,
+    yVertexMin = -0.5,
+    yVertexMax = 0.5,
+    zVertexMin = -35,
+    zVertexMax = 35 )
+
+
+genpileup = PoissonPileUp(name="Pileup", 
+    Filename="/afs/cern.ch/user/v/vavolkl/public/example_MyPythia.dat", 
```

Comment:
Can you point this to the fcc-sw dir:
`/afs/cern.ch/exp/fcc/sw/0.6/testsamples/example_MyPythia.dat`